tests/tck/features/basic/data.feature

```diff
@@ -0,0 +1,53 @@
+Feature: data
+
+  Background: Prepare space
+    Given a graph with space named "nba"
+
+  Scenario: list, set, map
+    When executing query:
+      """
+      RETURN size(LIST[]) AS a, size(SET{}) AS b, size(MAP{}) AS c
+      """
+    Then the result should be, in any order, with relax comparison:
+      | a | b | c |
+      | 0 | 0 | 0 |
+    When executing query:
+      """
+      RETURN 1 IN LIST[] AS a, "Tony" IN SET{} AS b, "a" IN MAP{} AS c
```

Comment:
could you add some nested cases， such as MAP{a: LIST[1,2], b: SET{1,2,1}, c: "hee"}

Reply:
It's ok